docs/boundary-services-a-to-z.adoc

```diff
@@ -7,8 +7,8 @@ The virtual hardware making up SoftNPU is depicted in the diagram below.
 
 image::plumbing.png[]
 
-The `softnpu` zone will be configured and launched during the
-`create_virtual_hardware.sh` script.
+The `softnpu` zone will be configured and launched during
+`cargo xtask virtual-hardware create`.
```

Comment:
If I'm on a bench gimlet where I have copied over my Omicron build, will this mean that
I now require `cargo` to be installed on that system as well?

I use basically the steps described here: https://github.com/oxidecomputer/meta/blob/master/engineering/omicron-on-sn06.adoc

Reply:
I'm going to read through the instructions here, but `cargo xtask` is a binary. So we should be able to use:

`cargo build --release -p xtask` to create this binary, and then we can move it around.

(Presumably we had to move the `create_virtual_hardware.sh` script around too?)

Reply:
> (Presumably we had to move the `create_virtual_hardware.sh` script around too?)

Correct.  Part of the copy steps include many things in the tools directory including that script.